src/launcher/Launcher.cc

```diff
@@ -233,6 +226,43 @@ void Launcher::startExecutable(bool asEditor) {
 	system(cmd.c_str());
 }
 
+bool Launcher::resolutionIsInvalid() {
+	return resolutionXInput->value() < 640 || resolutionYInput->value() < 480;
+}
+
+void Launcher::update(bool changed, Fl_Widget *widget) {
+	// not found warnings
+	if (check_if_relative_path_exists(dataDirectoryInput->value(), "Data", true)) {
+		dataDirNotFound->hide();
+		if (check_if_relative_path_exists(dataDirectoryInput->value(), "Data/Editor.slf", true)) {
```

Comment:
I would only show that when the editor button clicks. For most of the people who don't use the editor is is irrelevant.

Reply:
I like consistency, if the editor button only shows a warning when clicked, then the play button should do the same.

Reply:
Done.